smart-contracts/assembly/contracts/dns/dns.ts

```diff
@@ -228,6 +244,42 @@ function addToOwnerList(owner: Address, websiteName: string): void {
   );
 }
 
+/**
+ * Deletes a website name from the list of the given owner.
+ *
+ * @param websiteName - The name of the website to delete.
+ */
+function deleteFromOwnerList(websiteName: string): void {
+  const ownerbinary = owner(new Args().add(websiteName).serialize());
+  const ownerAddr = new Address(new Args(ownerbinary).nextString().unwrap());
+  const ownerListKey = ownerKey(ownerAddr);
+  const oldList = getOwnerWebsiteList(ownerAddr);
+  const oldListArray = oldList.split(',');
+
+  // Find the index of the websiteName in the old list array
+  const index = oldListArray.indexOf(websiteName);
+
+  // Check if the websiteName exists in the owner's list
+  if (index === -1) {
+    triggerError('WEBSITE_NOT_FOUND');
+    return;
+  }
+
+  // Remove the websiteName from the old list array
+  oldListArray.splice(index, 1);
```

Comment:
can you write a test case with 3 records, remove the 2, assert that the 2 others still exist?